tests/rust_namer_test.fbs

```diff
@@ -16,12 +16,20 @@ table PlayerInputChange {}
 
 union GameMessage
 {
-    PlayerStatEvent,
-    PlayerSpectate,
+  PlayerStatEvent,
+  PlayerSpectate,
 	PlayerInputChange
 }
 
 table GameMessageWrapper
 {
 	Message:GameMessage;
-}
\ No newline at end of file
+}
+
+struct PossiblyReservedWords
+{
+  follow: float;
```

Comment:
can you leave a comment that size and alignment are not reserved but they once were (and refer to idl_gen_rust for the current list)

Reply:
Added a comment to that effect.